lib/workspace.js

```diff
@@ -106,11 +106,17 @@ module.exports = {
             workspace_id: workspace_id,
             action: action,
         };
+
         const res = await fetch(`http://${workspace_host.hostname}/`, {
             method: 'post',
             body: JSON.stringify(postJson),
             headers: {'Content-Type': 'application/json'},
         });
+        if (action === 'grade') {
+            const dest = fs.createWriteStream('test.tgz');
```

Comment:
Maybe `test.tgz` is not the best name here? Could we call this `gradedFiles.tgz` to fit the purpose better?

Reply:
Changed.